Sources/SentryCrash/Recording/Monitors/SentryCrashMonitor_AppState.c

```diff
@@ -277,6 +284,18 @@ saveState(const char *const path)
         != SentryCrashJSON_OK) {
         goto done;
     }
+
+    // Follow the pattern of crashedLastLaunch to only store the calculated value if the app
+    // crashed.
+    double durationFromCrashStateInitToLastCrash = 0;
+    if (g_state.crashedThisLaunch) {
+        durationFromCrashStateInitToLastCrash = timeSince(g_crashstate_initialize_time);
+    }
+    if ((result = sentrycrashjson_addFloatingPointElement(&JSONContext,
+             kKeyDurationFromCrashStateInitToLastCrash, durationFromCrashStateInitToLastCrash))
+        != SentryCrashJSON_OK) {
+        goto done;
+    }
```

Comment:
I did not get this comment "only store ... if the app crashed". the code should not be like this then:

```suggestion
    // Follow the pattern of crashedLastLaunch to only store the calculated value if the app
    // crashed.
    double durationFromCrashStateInitToLastCrash = 0;
    if (g_state.crashedThisLaunch) {
        durationFromCrashStateInitToLastCrash = timeSince(g_crashstate_initialize_time);
        if ((result = sentrycrashjson_addFloatingPointElement(&JSONContext,
                 kKeyDurationFromCrashStateInitToLastCrash, durationFromCrashStateInitToLastCrash))
            != SentryCrashJSON_OK) {
            goto done;
        }
    }
```

Reply:
I rephrased it. Sorry this was confusing, thanks for pointing it out.